packages/@webex/webex-core/src/lib/services-v2/interceptors/server-error.js

```
@@ -0,0 +1,48 @@
+import {Interceptor} from '@webex/http-core';
+import WebexHttpError from '../../webex-http-error';
+/**
+ * Changes server url when it fails
+ */
+export default class ServerErrorInterceptor extends Interceptor {
+  /**
+   * @returns {HAMessagingInterceptor}
+   */
+  static create() {
+    // eslint-disable-next-line no-invalid-this
+    return new ServerErrorInterceptor({webex: this});
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)